app/models/album.rb

```diff
@@ -0,0 +1,2 @@
+class Album < Medium
+end
```

Comment:
This would be where you could add in those scopes mentioned in earlier comment.

Example:
scope :best, -> (num) { order('upvotes DESC').limit(num)}